### ISSUE TITLE
failed import in zoom.py on Python 3.9

### ISSUE THREAD

**Issue by emolter** [keflavich/image_registration#50]
[This line](https://github.com/keflavich/image_registration/blob/ed3ea3b2fc8aa1f014e6adbeee1c292a1a403f87/image_registration/fft_tools/zoom.py#L4C33-L4C33) causes an import error in Python 3.9 when trying to import chi2_shift.  Simply removing it in my local Conda installation allows me to run chi2_shift, but I'm not sure what it was originally used for.

By the way, would you please be willing to publish a release of e.g. 0.2.7 soon?  I've been building [some packages](https://github.com/emolter/pylanetary) that rely on your (amazing) tools, but pypi will not allow me to publish a release with dependencies that download directly from a GitHub page instead of from pypi.  I'd be happy to help by submitting a PR if there are outstanding issues that need to be resolved before you can make a release.

**Comment by keflavich** [comment]
@emolter If you want to PR fixes like this, they're welcome.  The link you posted went to the anchor `#L4C33-L4C33`, which doesn't actually bring me to a line.  I'll try to hunt down the issue though.

Also, yes, if I can get tests passing (...I don't have much hope for getting CI running), I'll issue a new release

**Comment by keflavich** [comment]
I don't see any import error myself.  Could you provide a traceback?
```
$ python -c "import sys, image_registration; print(sys.version, image_registration.__version__); from image_registration import chi2_shift"
3.9.16 | packaged by conda-forge | (main, Feb  1 2023, 21:38:11)
[Clang 14.0.6 ] 0.2.7.dev22+ged3ea3b
```

**Comment by emolter** [comment]
I will go ahead and submit PRs for stuff like this in the future, sorry about that - I'm still getting used to the contribution workflow.  It looks like you already solved this, but here is the traceback for reference:

    from image_registration.chi2_shifts import chi2_shift
../../../../anaconda/envs/py39/lib/python3.9/site-packages/image_registration/__init__.py:14: in <module>
    from .cross_correlation_shifts import cross_correlation_shifts, cross_correlation_shifts_FITS
../../../../anaconda/envs/py39/lib/python3.9/site-packages/image_registration/cross_correlation_shifts.py:2: in <module>
    from .fft_tools import correlate2d
../../../../anaconda/envs/py39/lib/python3.9/site-packages/image_registration/fft_tools/__init__.py:5: in <module>
    from .upsample import dftups,upsample_image
../../../../anaconda/envs/py39/lib/python3.9/site-packages/image_registration/fft_tools/upsample.py:5: in <module>
    from . import zoom
../../../../anaconda/envs/py39/lib/python3.9/site-packages/image_registration/fft_tools/zoom.py:4: in <module>
    from matplotlib import docstring
E   ImportError: cannot import name 'docstring' from 'matplotlib' (/Users/emolter/anaconda/envs/py39/lib/python3.9/site-packages/matplotlib/__init__.py)

**Comment by keflavich** [comment]
Yep, looks like https://github.com/keflavich/image_registration/pull/52 fixed it - so PRs like that are totally welcome.  I now have #53 open to try to fix CI, but I can only look at it... post-JWST-deadline
